packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -487,11 +540,40 @@ export function makeVirtualObjectManager(
       if (initializing) {
         cache.remember(innerSelf);
       }
-      const representative = Far(tag, actualize(wrappedData));
+      const self = actualize(wrappedData);
+      let representative;
+      let exposedRepresentative;
+      const facetiousness = assessFacetiousness(self);
+      switch (facetiousness) {
+        case 'one': {
+          representative = Far(tag, self);
+          vrm.checkOrAcquireFacetNames(kindID, null);
+          exposedRepresentative = representative;
+          break;
+        }
+        case 'many': {
+          exposedRepresentative = {};
+          representative = [];
+          const facetNames = Object.getOwnPropertyNames(self).sort();
```

Comment:
Since you've already validated that these are only strings, a simple `.sort()` works. Good. No change suggested.